hw/ip/pinmux/data/pinmux_fpv_testplan.hjson

```diff
@@ -0,0 +1,119 @@
+{
+  name: "pinmux"
+  import_testplans: ["hw/dv/tools/testplans/fpv_csr_testplan.hjson"]
+  entries: [
+    {
+      name: InSel0_A
+      desc: "When register periph_insel sets to 0, the corresponding mio_to_periph_o should be 0."
```

Comment:
I would maybe reword `sets` as `is set to` and use the word `must` instead of `should` since this is a strict requirement

Reply:
Thanks Michael, sounds good, will update the rest of the docs.